src/benchmarks/rpmem_persist.cpp

```diff
@@ -232,7 +233,7 @@ rpmem_op(struct benchmark *bench, struct operation_info *info)
 		assert(info->worker->index < mb->nlanes[r]);
 
 		ret = rpmem_persist(mb->rpp[r], offset, len,
-				    info->worker->index);
+				    info->worker->index, 0 /* XXX */);
```

Comment:
We shall also have a configuration option that allows us to run benchmarks with PMEM_F_RELAXED flag